src/features/ajax/instrument/index.js

```diff
@@ -34,6 +34,29 @@ export class Instrument extends InstrumentBase {
     this.dt = new DT(agentIdentifier)
 
     this.handler = (type, args, ctx, group) => handle(type, args, ctx, group, this.ee)
+
+    // this is a best (but imperfect) effort at capturing AJAX calls that may have fired before the agent was instantiated
+    // this could happen because the agent was "improperly" set up (ie, not at the top of the head) or
+    // because it was deferred from loading in some way -- e.g. 'deferred' script loading tags or other lazy-loading techniques
+    //
+    // it is "imperfect" because we cannot capture the following with the API vs wrapping the events directly:
+    // * requestBodySize - (txSize -- request body size)
+    // * method - request type (GET, POST, etc)
+    // * callbackDuration - (cbTime -- sum of resulting callback time)
+    try {
+      const initiators = { xmlhttprequest: 'xhr', fetch: 'fetch', beacon: 'beacon' }
+      globalScope?.performance?.getEntriesByType('resource').forEach(resource => {
+        if (Object.keys(initiators).includes(resource.initiatorType) && resource.responseStatus !== 0) {
```

Comment:
```suggestion
        if (resource.initiatorType in initiators && resource.responseStatus !== 0) {
```
^is a much faster operation btw

Reply:
<img width="848" alt="Screenshot 2023-08-23 at 10 34 55 PM" src="https://github.com/newrelic/newrelic-browser-agent/assets/24296585/918ad1a5-036c-4015-8265-8c6a6040c766">

Much* is subjective i guess when we are talking about 3 keys -- it's negligible, but we can go with this